spec/features/user_follows_and_unfollows_another_user_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+feature "A User follows another user" do
+  scenario "and sees that they follow them" do
```

Comment:
This sentence needs more nouns maybe. Something like "and the follower sees the followed user's recommendations" or is that out of scope (e.g. we are just showing who you follow)?